.jenkins/pytorch/build.sh

```diff
@@ -192,9 +192,20 @@ if [[ "$BUILD_ENVIRONMENT" == *-bazel-* ]]; then
 
   get_bazel
 
-  tools/bazel build --config=no-tty //...
+  # Bazel build is run on a linux.2xlarge (or c5.2xlarge) runner with 8 CPU and 16GB of memory.
+  # The build job sometimes fails with 'runner lost communication with the server' flaky error
+  # which indicates that something there crashes the runner process. The most likely reason is
+  # that the build runs out of memory. So trying to follow https://bazel.build/docs/user-manual
+  # to limit the memory the CPU and memory usage, so that we will know even if it crashes with
+  # OOM error instead of crashing the runner and losing all the logs.
+  #
+  # Leave 1 CPU free and use only up to 80% of memory
+  BAZEL_MEM_LIMIT="--local_ram_resources=HOST_RAM*.8"
+  BAZEL_CPU_LIMIT="--local_cpu_resources=HOST_CPUS-1"
+
+  tools/bazel build --config=no-tty "${BAZEL_MEM_LIMIT}" "${BAZEL_CPU_LIMIT}" //...
```

Comment:
Shouldn't those quotation marks be removed?

Seems like this would expand to
`tools/bazel build --config=no-tty "--local_ram_resources=HOST_RAM*.8" "--local_cpu_resources=HOST_CPUS-1" //...`

Reply:
Huh, it expanded to `tools/bazel build --config=no-tty '--local_ram_resources=HOST_RAM*.8' --local_cpu_resources=HOST_CPUS-1 //...` in your test. 

https://pipelines.actions.githubusercontent.com/serviceHosts/7d146c05-69c3-4c20-a0e7-818111670117/_apis/pipelines/1/runs/2540998/signedlogcontent/36?urlExpires=2023-01-12T18%3A26%3A11.1264068Z&urlSigningMethod=HMACV1&urlSignature=66%2BjAfRNiidgd29JFUgqwa%2FFsVYg3rirQwllhEOMKcQ%3D

Reply:
Yeah, I have confirmed that the command work with the quotation, I need to keep them to satisfy the linter god :P